src/components/FilterPanel.jsx

```diff
@@ -73,9 +93,43 @@ export default function FilterPanel({ onClose }) {
             <Typography>{name}</Typography>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      {filtersChanged && (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            mt: 2,
+          }}>
+          <WarningAmberOutlinedIcon color="warning" />
+          <Typography>Your changes have not yet been applied</Typography>
```

Comment:
Nit: are you able to add a space here?
```suggestion
          <Typography> Your changes have not yet been applied</Typography>
```

Reply:
Also a period is missing at the end here.

Reply:
doesn't seem to be adding a space, but I can include a margin of 8px